### ISSUE TITLE
Web Accessibility

### ISSUE THREAD

**Issue by 01es** [fieldenms/tg#291]
[Web Accessibility](https://www.w3.org/WAI/intro/accessibility.php) is becoming of high importance, especially in the context of application for the Health industry. Polymer already provides some support in building accessible web components, and this support is going to be improved over time.

This issue should be used for further research and experimentation with the purpose of gaining the necessary knowledge and best practices for supporting Web Accessibility as part of TG. It is very important to understand that Web Accessibility is not targeted only at people with disabilities, but at providing more usable applications in general (e.g. focus and tab index handling, keyboard support).

There is already a [Gold Standard](https://github.com/webcomponents/gold-standard/wiki) for web components, which should be used as a set of guide lines when building web components.
